src/components/Common/FilePreviewDialog.tsx

```
@@ -170,6 +183,67 @@
     onClose?.();
   };
 
+  const handleRotate = (rotation: number) => {
+    setFileState((prev: any) => ({
+      ...prev,
+      rotation: prev.rotation + rotation,
+    }));
+  };
+
+  function getRotationClass(rotation: number) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getRotationClass'. Consider to delete it or rename it.

'getRotationClass' is defined here:


(lint/suspicious/noRedeclare)